helm/prometheus-operator-app/values.yaml

```diff
@@ -2709,6 +2709,3 @@ prometheus:
     ## https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#podmetricsendpoint
     ##
     # podMetricsEndpoints: []
-
-crds:
-  install: true
```

Comment:
This is a lie, crds are in a different chart now.

Reply:
The cake is a lie

Reply:
![image](https://user-images.githubusercontent.com/6536819/150371945-a53e7bc9-a780-4b04-aaf2-5020fbad6dbf.png)